decidim-assemblies/app/cells/decidim/assemblies/content_blocks/main_data_cell.rb

```diff
@@ -20,10 +23,25 @@ def title_text
           t("title", scope: "decidim.assemblies.assemblies.show")
         end
 
-        def description_text
+        def short_description_text
           decidim_sanitize_editor_admin translated_attribute(short_description)
         end
 
+        def description_text
+          [decidim_sanitize_editor_admin(translated_attribute(description)), extra_attributes].compact_blank.join("\n")
+        end
+
+        def extra_attributes
+          %w(purpose_of_action internal_organisation composition).filter_map do |attribute|
+            if (text = translated_attribute(send(attribute))).present?
+              [
+                content_tag(:h3, class: "h4") { t(attribute, scope: "activemodel.attributes.assembly") },
+                decidim_sanitize_editor_admin(text)
+              ].join("\n")
+            end
```

Comment:
I prefer a guard clause here:

```suggestion
            next if (text = translated_attribute(send(attribute))).blank?

            [
              content_tag(:h3, class: "h4") { t(attribute, scope: "activemodel.attributes.assembly") },
              decidim_sanitize_editor_admin(text)
            ].join("\n")
```